workflow/rules/enrichment.smk

```diff
@@ -40,10 +40,29 @@ rule spia:
         "../envs/spia.yaml"
     log:
         "logs/tables/pathways/{model}.spia-pathways.log",
-    threads: 16
+    threads: 38
     script:
         "../scripts/spia.R"
 
+rule plot_spia_pathways:
```

Comment:
This rule and script can be removed as we have converted the contents into datavzrd.

Reply:
Noted. removed